src/project/internal/projectconfiguration.h

```diff
@@ -74,9 +73,13 @@ class ProjectConfiguration : public IProjectConfiguration
     bool shouldAskSaveLocationType() const override;
     void setShouldAskSaveLocationType(bool shouldAsk) override;
 
-    bool isCloudProject(const io::path_t& projectPath) const override;
+    bool isOldCloudProject(const io::path_t& projectPath) const override;
```

Comment:
Why do we need two of them in the interface? It's better to keep only isCloudProject and hide all the details behind its implementation

Reply:
I cleaned this up a little bit, and ensured that it also works well when opening "legacy" cloud scores (from the old cloud scores folder from 4.0)